v1/counterparty/counterparty.gunk

```diff
@@ -0,0 +1,404 @@
+	//                         Schema: openapiv2.Schema{
+	//                                 JSONSchema: openapiv2.JSONSchema{
+	//                                         Ref: "#/definitions/counterpartyCreateCounterPartyResponse",
+	//                                 },
+	//                         },
+	//                 },
+	//         },
+	//         Security: []openapiv2.SecurityRequirement{
+	//                 {
+	//                         SecurityRequirement: map[string]openapiv2.SecurityRequirement_SecurityRequirementValue{
+	//                                 "OAuth2": openapiv2.SecurityRequirement_SecurityRequirementValue{
+	//                                         Scope: []string{
+	//                                                 "write",
+	//                                         },
+	//                                 },
+	//                         },
+	//                 },
+	//         },
+	// }
+	CreateCounterParty(CreateCounterPartyRequest) CreateCounterPartyResponse
+
+	// GetCounterPart retrieves the details of an counter party, selected by its id.
+	//
+	// +gunk http.Match{
+	//         Method: "GET",
+	//         Path:   "/v1/counterparty/{ID}",
+	// }
+	// +gunk openapiv2.Operation{
+	//         Tags:        []string{"CounterParty"},
+	//         Description: "Retrieves all data from an counter party selected by the supplied counter_party_id.",
+	//         Summary:     "Retrieve an counter party",
+	//         Responses: map[string]openapiv2.Response{
+	//                 "200": openapiv2.Response{
+	//                         Description: "Request executed successfully.",
+	//                         Schema: openapiv2.Schema{
+	//                                 JSONSchema: openapiv2.JSONSchema{
+	//                                         Ref: "#/definitions/counterpartyCounterParty",
+	//                                 },
+	//                         },
+	//                 },
+	//                 "404": openapiv2.Response{
+	//                         Description: "Returned when the resource is not found.",
+	//                 },
+	//         },
+	//         Security: []openapiv2.SecurityRequirement{
+	//                 {
+	//                         SecurityRequirement: map[string]openapiv2.SecurityRequirement_SecurityRequirementValue{
+	//                                 "OAuth2": openapiv2.SecurityRequirement_SecurityRequirementValue{
+	//                                         Scope: []string{
+	//                                                 "read",
+	//                                         },
+	//                                 },
+	//                         },
+	//                 },
+	//         },
+	// }
+	GetCounterParty(GetCounterPartyRequest) CounterParty
+
+	// GetCounterParties returns a list containing up to 20 counter parties.
+	//
+	// +gunk http.Match{
+	//         Method: "GET",
+	//         Path:   "/v1/counterparties",
```

Comment:
Why above(L229 and L264) is counterparty and here is counterparties?

Reply:
@nkristianto you can say `bank` -> `banks` (for get all request), but `party` will become `parties`. you can't say `partys`

Reply:
for me, that's not consistent, because one using party and another use parties. @jwangsadinata wdyt